src/network/message.rs

```diff
@@ -188,12 +198,16 @@ impl NetworkMessage {
             NetworkMessage::Alert(_)    => "alert",
             NetworkMessage::Reject(_)    => "reject",
             NetworkMessage::FeeFilter(_) => "feefilter",
+            NetworkMessage::Unknown { .. } => "unknown",
         }
     }
 
     /// Return the CommandString for the message command.
     pub fn command(&self) -> CommandString {
-        self.cmd().into()
+        match *self {
+            NetworkMessage::Unknown { command: ref c, .. } => c.clone(),
```

Comment:
Here we return internal command representation, unlike to `cmd` method above (see my previous comment)

Reply:
Yeah so this one returns CommandString with a `Cow::Owned` because it's an unknown command, but we use `cmd` to craft `Cow::Borrowed` CommandStrings for outgoing messages.